packages/contracts/test/setup.ts

```diff
@@ -0,0 +1,49 @@
+import { MockProvider, Fixture } from 'ethereum-waffle'
+import { Wallet } from 'ethers'
+import { waffle } from 'hardhat'
+
+type FixtureLoader = ReturnType<typeof waffle.createFixtureLoader>
+interface FixtureReturns {
+  provider: MockProvider,
+  wallet: Wallet,
+  other: Wallet,
+  another: Wallet,
+}
+
+let loadFixture: ReturnType<typeof setupOnce> | undefined
+export function setupFixtureLoader () {
+  if (!loadFixture) {
+    loadFixture = setupOnce()
+  }
+  return loadFixture
+}
+
+type CurrentLoader = { loader: FixtureLoader, returns: FixtureReturns, fixture: Fixture<any> }
+
+function setupOnce () {
```

Comment:
```suggestion
function setupOnce() {
```
Update our linter config 😅